### DISCUSSION TITLE
Unable to render rich display

### DISCUSSION THREAD

**Discussion by N-Uday** [community/community#42258] [New to GitHub]
If i try to upload/open any file i receive this message unable to render rich display,
Even if i try to open any file as raw, then  i will get raw.githubusercontent.com took too long to respond.

Can anyone help me with this problem..?
![Screenshot (461)](https://user-images.githubusercontent.com/120168123/208833354-b5f6260f-2f49-47be-a2da-be34bef156cd.png)

**Comment by DSDmark** [comment]
Hi @N-Uday,

Not sure about that. But, There are several possible causes for the error.

1. Check the file type: Some file types are not supported by GitHub's rich display feature.
2. Check the file size: If the file is very large, it may take a long time to load or may not load at all.
3. Check your network connection: If you are experiencing issues loading the file, it may be due to a problem with your internet connection.
4. Check the file's content: If the file contains invalid or malicious content, GitHub may not be able to render it.

**Comment by sambhavjain98** [comment]
Hello @N-Uday , 
I think the issue is quite recent as per the below link, with a change in DNS settings, the content gets rendered successfully. This is for anyone else who is facing this issue presently. 
https://github.com/community/community/discussions/41020
Use the below link to change the DNS settings
https://www.youtube.com/watch?v=1MMnUBsUczY&feature=youtu.be

Hope it helps 👍

**Reply by GaneshJalakam** [reply]
@sambhavjain98 ,
Dude, thanks for your help. I changed the DNS settings and got the issue resolved.

**Reply by Rdx-4141** [reply]
Thanks mate, It worked for me also.

**Comment by jose19961975** [comment]
By turning everything off in the house so in bedroom 2 can reboot lmao

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
